Makefile

```diff
@@ -122,19 +123,22 @@ logs.%: ## Shows the logs of the given docker service. Example: make logs.datast
 restart.%: ## Restarts the given docker service. Example: make restart.datastore
 	docker compose restart $*
 
-run_tests.prettier: ## Formats the code using prettier.
+run_tests.prettier: ## Runs the prettier checks
 	docker compose run --no-deps --entrypoint "npx prettier --check ." dev-server
 
 run_tests.third_party_size_check: ## Runs the third party size check
 	docker compose run --no-deps --entrypoint "python -m scripts.third_party_size_check" dev-server
 
 run_tests.lint: ## Runs the linter tests
-	docker compose run --no-deps --entrypoint "/bin/sh -c 'git config --global --add safe.directory /app/oppia && python -m scripts.linters.run_lint_checks $(PYTHON_ARGS)'" dev-server || $(MAKE) stop
+	docker compose run --no-deps --entrypoint "/bin/sh -c 'git config --global --add safe.directory /app/oppia && python -m scripts.linters.run_lint_checks $(PYTHON_ARGS)'" dev-server
```

Comment:
Here and elsewhere, why are we removing the finishing `make stop` command? Don't we want to shut down the containers once the tests complete?

Reply:
I have removed `make stop` from these commands that used `docker compose` since the container gets terminated without `make stop`. I'll look into the remaining commands and update this thread.